### PR TITLE
Fixes #9668: Document the Audit mode

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/9668

### CODE REVIEW

**Comment by jooooooon** [comment]
Ping @peckpeck or @fanf for review - this needs to get into official documentation ASAP!

File: 22_configuration_management/40_policy_mode.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+=== Policy Mode (Audit/Enforce)
+
+Rudder 4.0 includes a policy mode setting, that allows two distinct behaviors:
+
+* *Audit*: Test if the system is in the desired state, and report about it
+* *Enforce*: Test if the system is in the desired state, if not, try to act to get to this state, and report about actions taken and final state
+
+This mode can be set:
+
+* Globally on the Rudder root server. In this can there are two options: allow to override this mode on specific items, or use the global configuration evrywhere.
```
Comment (peckpeck) [inline 22_configuration_management/40_policy_mode.txt]: in this case

Comment (peckpeck) [inline 22_configuration_management/40_policy_mode.txt]: everywhere

File: 22_configuration_management/40_policy_mode.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* On a node.
+
+A lot of attention and several safeguards have been put in place to ensure that if you choose to use "Audit"
+for a target, nothing will be changed on the node for that target (except Rudder's own configuration under `/var/rudder`), and only some harmless
+commands will be run (like listing installed packages or refreshing package lists).
+
+From a technical point of view, the configuration and compliance assessing of Rudder was greatly improved in Rudder
+4.0 to accomodate the new Audit policy mode:
+
+* The expected configurations calculated for each nodes are more precise, and as a bonus it should also assure a better resilience of results.
```
Comment (peckpeck) [inline 22_configuration_management/40_policy_mode.txt]: More precise than what ? This is not a changelog so there is no reference point

File: 22_configuration_management/40_policy_mode.txt [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+* Nodes are fully aware of exactly what directives need to be executed in Audit or in Enforce mode, and the "rudder agent" command line has been enhanced to let you see the result in a glimpse.
```
Comment (peckpeck) [inline 22_configuration_management/40_policy_mode.txt]: with a glimpse

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by ncharles** [comment]
This is good for me also. Tagging as ready for merge (i didn't see the commit modified first, that's why I added and removed the tag)

**Pull request merged**